provider/cmd/pulumi-resource-meraki/bridge-metadata.json

```diff
@@ -7792,5 +7792,525 @@
             }
         }
     },
-    "auto-settings": {}
+    "auto-settings": {},
+    "ids": {
+        "resources": {
+            "meraki_administered_licensing_subscription_subscriptions_bind": [
+                "subscription_id"
+            ],
+            "meraki_devices_appliance_radio_settings": [
+                "serial"
+            ],
```

Comment:
I'm surprised these are all new, was the manual `ComputeID` field approach not working at all? Did I just fail to check in the generated change?

Reply:
This is an artifact of `tfbridge.MustTraverseProperties`. Doing a full traversal of every schema property hurts startup-time, so `tfbridge.MustTraverseProperties` keeps track of the properties it needs to make changes on and when the provider boots, it only loads as much as it needs to, visiting properties that have changes (recorded here) and ignoring properties that don't need to be visited.

Reply:
The manual approach was working fine, accept that each new resource would have required manual intervention. You didn't miss checking in a generated change.